pybind11/__init__.py

```diff
@@ -2,27 +2,8 @@
 
 
 def get_include(user=False):
-    from distutils.dist import Distribution
-    import os
+    from os import pardir
+    from os.path import abspath, dirname, join
     import sys
-
-    # Are we running in a virtual environment?
-    virtualenv = hasattr(sys, 'real_prefix') or \
-        sys.prefix != getattr(sys, "base_prefix", sys.prefix)
-
-    if virtualenv:
-        return os.path.join(sys.prefix, 'include', 'site',
-                            'python' + sys.version[:3])
-    else:
-        dist = Distribution({'name': 'pybind11'})
-        dist.parse_config_files()
-
-        dist_cobj = dist.get_command_obj('install', create=True)
-
-        # Search for packages in user's home directory?
-        if user:
-            dist_cobj.user = user
-            dist_cobj.prefix = ""
-        dist_cobj.finalize_options()
-
-        return os.path.dirname(dist_cobj.install_headers)
+    prefix = abspath(join(dirname(__file__), pardir, pardir, pardir, pardir))
+    return join(prefix, 'include', 'python' + sys.version[:3])
```

Comment:
Hm. I guess this has the following issues:
- Won't work on Windows
- Won't work with pymalloc installations, where the directory is called "./include/python36m/"